Dockerfile

```diff
@@ -100,6 +92,13 @@ RUN wget -q https://cmake.org/files/v3.10/cmake-3.10.0.tar.gz -O- \
     | tar -C /tmp -xz && cd /tmp/cmake-3.10.0/ && ./bootstrap && \
     make && make install && cd && rm -rf /tmp/cmake-3.10.0
 
+# Install ARM binary toolchain
+RUN mkdir -p /opt && \
+        wget -q https://armkeil.blob.core.windows.net/developer/Files/downloads/gnu-rm/7-2018q2/gcc-arm-none-eabi-7-2018-q2-update-linux.tar.bz2 -O- \
```

Comment:
You can use a more "official" base url: `https://developer.arm.com/-/media/Files/downloads/gnu-rm/7-2018q2/gcc-arm-none-eabi-7-2018-q2-update-linux.tar.bz2`.

For IoT-LAB, we use the following full command:
```
wget -nv -O - https://developer.arm.com/-/media/Files/downloads/gnu-rm/7-2018q2/gcc-arm-none-eabi-7-2018-q2-update-linux.tar.bz2 | tar -jxf -
```
With `tar -C /opt -jxf -`, it should be the same.

Reply:
So far, the URL used there is the one which redirects from the ARM website. Though both are the same and equally official.

Reply:
Changed.